src/site/components/merger-crosslinks.html

```diff
@@ -30,11 +30,12 @@ <h3 class="va-nav-linkslist-heading">{{ group.heading }}</h3>
     {% endif %}
 
     {% if group.links.length > 0 %}
+    {% assign group_heading = group.heading %}
       <ul class="va-nav-linkslist-list">
       {% for link in group.links %}
         {% if link.url != empty %}
           <li>
-            <a href="{{link.url}}">
+            <a href="{{link.url}}" onClick="recordEvent({ event: 'nav-linkslist', 'links-list-header':  '{{ group_heading }}'  });">
```

Comment:
Can you use `group.heading` here instead of assigning another variable?

Reply:
I tried using `group.heading` but for scoping reasons it wasn't working

Reply:
Liquid. :man_facepalming: 
:man_shrugging: